modules/ROOT/pages/_partials/intro-con.adoc

```diff
@@ -26,14 +28,16 @@ The {webapp} presents links to the applications available in a {PRODUCT_SHORT} c
 
 * *Red Hat Fuse Online*, a facility to obtain data from an application or service, operate on that data, and then send the data to another application or service without writing code.
 
-* *Red Hat Single Sign-On*, a single sign-on solution for web apps and RESTful web services.
+* *Push Notifications*, a service to push notifications to Android and iOS devices.
```

Comment:
```suggestion
* *Push Notifications*, a service to push messages to Android and iOS devices.
```
Minor pickiness here but should we be saying 'notifications' twice in the same sentence?

Reply:
don't think we can call them messages